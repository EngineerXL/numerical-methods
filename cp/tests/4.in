130318350226079115269304
2699128795112291614751982
