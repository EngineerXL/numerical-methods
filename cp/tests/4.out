351746011626738557950203348393669616075097760528
